lib/std/os.zig

```diff
@@ -4707,11 +4707,13 @@ pub fn munmap(memory: []align(mem.page_size) const u8) void {
 
 pub const MSyncError = error{
     UnmappedMemory,
+    HitRegionImmutable,
```

Comment:
I suggest changing the error name to `ImmutableMemory` (which mirrors the existing `UnmappedMemory` name).  However the Zig team might have other ideas on how to (re)name this error, so you might want to wait and see if there are other suggestions.